include/ignition/math/detail/Box.hh

```diff
@@ -113,6 +121,159 @@ T Box<T>::Volume() const
   return this->size.X() * this->size.Y() * this->size.Z();
 }
 
+//////////////////////////////////////////////////
+template <typename T>
+std::vector<std::pair<Triangle3<T>, T>> TrianglesInPlane(
```

Comment:
this helper function could use a doc-string with a high-level explanation of what it does